src/models/tools/geometry/jxg-board.ts

```diff
@@ -85,6 +86,12 @@ export function guessUserDesiredBoundingBox(board: JXG.Board) {
   return [xMin + xBufferRange, yMax - yBufferRange, xMax - xBufferRange, yMin + yBufferRange];
 }
 
+function getAxisUnits(changeProps?: JXGProperties) {
```

Comment:
I'd be inclined to call this `getAxisUnitsFromProps()` just to be clear that it's not pulling from the board or anywhere else.

Reply:
I was considering that, +1